go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -126,15 +131,62 @@ func (se *Engine) InitDBConfig(cp dbconfigs.Connector) {
 	se.cp = cp
 }
 
+// syncVTDatabase is called either the first time a primary starts or on consequent loads to possibly upgrade to a
+// new Vitess version. This is the only entry point into the sidecardb module to get the _vt database to the desired
+// schema for the running Vitess version.
+// There is some extra logging in here which can be removed in a future version (>v16) once the new schema init
+// functionality is stable.
+func (se *Engine) syncVTDatabase(ctx context.Context, conn *dbconnpool.DBConnection) error {
```

Comment:
Another case where I think syncSidecarDB() is better.

Reply:
done